### ISSUE TITLE
[Docs]: Documentation for older versions of react-router in the 6.x branch is not available.

### ISSUE THREAD

**Issue by DamianPereira** [remix-run/react-router#10108]
### Describe what's incorrect/missing in the documentation

I have a project using react router 6.2.1, and I can not find the documentation for that version, I had to resort to Archive.org and changelogs to realize that a lot of the examples in the documentation were not working because of the APIs included in react router 6.4. I think the version selector should allow at least latest mayor version from 6.0 to 6.8, if not, the bare minimum I think would be latest before 6.4.

**Comment by DamianPereira** [comment]
I don't think this should be closed! Old documentation should be available somehow.

**Comment by ryanflorence** [comment]
Fixed, we were fetching the tarballs wrong when we switched our release tagging format, I just special cased it on the website, it's deploying now.

You should be able to visit any v6+ release when it's deployed, like https://reactrouter.com/6.2.1
